activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Disable automatic write protection on replicas.
+
+    Write protection is no longer automatically enabled for replicas. Write protection should be enabled by the database user settings.
```

Comment:
Can we expand with an example what "Write protection should be enabled by the database user settings." means?

Reply:
Good idea @rafaelfranca, I put together https://github.com/rails/rails/pull/42529

I am not sure how much detail to add since it is not only database specific, but also configuration specific. Depending on how your primary and replica are setup you might want a totally different configuration. Ideally you would setup a local database user to be very similar to how your production users are setup.